server/backend/housekeeping/housekeeping.go

```diff
@@ -49,19 +48,33 @@ func New(conf *Config) (*Housekeeping, error) {
 
 // RegisterTask registers task the housekeeping service.
 func (h *Housekeeping) RegisterTask(
```

Comment:
How about leaving this function `RegisterTask()` to just register one task, and make two `RegisterTask()`s (client deactivation, hard deletion)?